lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -256,6 +261,8 @@ def build_entity(self):
             name=self._module.params['name'],
             description=self._module.params['description'],
             comment=self._module.params['comment'],
+            import_=True if (self._module.params['state'] == 'import' and storage_type in ['iscsi', 'fcp']) else None,
```

Comment:
IINM For NFS storage domain we do not mention any indication whether it is a new storage domain or an imported one. The engine simply check if there is an existing storage domain, and if there is it imports it